src/uproot/behaviors/RNTuple.py

```diff
@@ -725,6 +871,107 @@ def arrays(
 
         return arrays
 
+    def _arrays_GDS(self, columns, entry_start=0, entry_stop=None):
+        """
+        Current GDS support is limited to nvidia GPUs. The python library kvikIO is
+        a required dependency for Uproot GDS reading which can be installed by
+        calling pip install uproot[GDS_cux] where x corresponds to the major cuda
+        version available on the user's system.
+        Args:
+            columns (list of str): Names of ``RFields`` or
+                aliases to convert to arrays.
+            entry_start (None or int): The first entry to include. If None, start
+                at zero. If negative, count from the end, like a Python slice.
+            entry_stop (None or int): The first entry to exclude (i.e. one greater
+                than the last entry to include). If None, stop at
+                :ref:`uproot.behaviors.TTree.TTree.num_entries`. If negative,
+                count from the end, like a Python slice.
+        """
+        #####
+        # Find clusters to read that contain data from entry_start to entry_stop
+        entry_start, entry_stop = (
+            uproot.behaviors.TBranch._regularize_entries_start_stop(
+                self.num_entries, entry_start, entry_stop
+            )
+        )
+        clusters = self.ntuple.cluster_summaries
+        cluster_starts = numpy.array([c.num_first_entry for c in clusters])
+        start_cluster_idx = (
+            numpy.searchsorted(cluster_starts, entry_start, side="right") - 1
+        )
+        stop_cluster_idx = numpy.searchsorted(cluster_starts, entry_stop, side="right")
+        cluster_num_entries = numpy.sum(
+            [c.num_entries for c in clusters[start_cluster_idx:stop_cluster_idx]]
+        )
+
+        # Get form for requested columns
+        form = self.to_akform().select_columns(columns, prune_unions_and_records=False)
+
+        # Only read columns mentioned in the awkward form
+        target_cols = []
+        container_dict = {}
+        uproot.behaviors.RNTuple._recursive_find(form, target_cols)
+
+        #####
+        # Read and decompress all columns' data
+        clusters_datas = self.GPU_read_clusters(
+            target_cols, start_cluster_idx, stop_cluster_idx
+        )
+        clusters_datas.decompress()
+        #####
+        # Deserialize decompressed datas
+        content_dict = self.Deserialize_decompressed_content(
```

Comment:
Rename this function to use [snake case](https://en.wikipedia.org/wiki/Snake_case) to be consistent with the other functions in the library.